components/movementsensor/dualgps/dualgps.go

```diff
@@ -264,7 +264,12 @@ func (dg *dualGPS) Orientation(ctx context.Context, extra map[string]interface{}
 }
 
 func (dg *dualGPS) Accuracy(ctx context.Context, extra map[string]interface{}) (*movementsensor.Accuracy, error) {
-	return nil, movementsensor.ErrMethodUnimplementedAccuracy
+	// RSDK-6389: for this driver, find the highest value VDOP and HDOP to show the worst accuracy in position
```

Comment:
Same comments as the previous PR: unmarked comments like this look like FYI, but I suspect this is actually a TODO, so I'd change the start to `// TODO[RSDK-6389]: ...`